app/controllers/concerns/response.rb

```diff
@@ -0,0 +1,5 @@
+module Response
+  def json_response(object, status = :ok)
+    render json: object, status:
+  end
+end
```

Comment:
- [ ] There is a slight error in this implementation that is making the serve output an error. Please kindly include a status code or remove it if you don't want it